pkg/cluster/spec/validate.go

```diff
@@ -794,6 +794,22 @@ func (s *Specification) validateTLSEnabled() error {
 	return nil
 }
 
+func (s *Specification) validatePdPodNames() error {
+	// check pdserver pod name
+	if len(s.PDServers) > 1 {
+		cnt := make(map[string]int)
```

Comment:
Prefer to use [StringSet](https://github.com/pingcap/tiup/blob/master/pkg/set/string_set.go):

```
names := set.NewStringSet()

for _, pd := range s.PDServers {
    if pd.Name == "" {
        continue
    }
    if names.Exist(pd.Name) { 
        return errors.Errorf("xxxxxx")
    }
    names.Insert(pd.Name)
}
```

Reply:
got it